cmd/maya-exporter/app/command/command.go

```diff
@@ -35,15 +55,22 @@ func AddControllerAddressFlag(cmd *cobra.Command, value *string) {
 		"Address of the Jiva volume controller.")
```

Comment:
If a different storage engine or casType is used - this message will be inaccurate.

Reply:
@kmova this flag will only be valid for jiva. We will not be using this flag for cstor because the plan is to read directly from the unix domain sockets.